src/adapter/src/coord/read_policy.rs

```diff
@@ -278,6 +278,27 @@ impl<T: TimelyTimestamp + Lattice> ReadHoldsInner<T> {
         since
     }
 
+    /// Returns the frontier at which this [ReadHoldsInner] is holding back the
+    /// since of the collection identified by `id`.
+    ///
+    /// This method is not meant to be fast, use wisely!
+    pub fn since(&self, desired_id: &GlobalId) -> Antichain<T> {
```

Comment:
`since` is maybe the wrong name? The returned frontier might be greater than the actual collection `since`. I think `least_valid_read` would be the perfect name, but that might cause collisions/confusion.

Reply:
I could call it `frontier()` :sweat_smile: . But I still think `since` is fine: it doesn't have to be the overall collection since, it's only what this ReadHolds happends to hold it at

Reply:
or `time()`?

Reply:
I decided to merge with `since()` because that's also what, for example, a persist `ReadHandle` calls the frontier at which it holds.

Reply:
Yeah, I've come around to this reasoning now too!